mlflow/utils/rest_utils.py

```diff
@@ -50,6 +50,11 @@ def request_with_ratelimit_retries(max_rate_limit_interval, **kwargs):
         time_left = max_rate_limit_interval
         sleep = 1
         while response.status_code == 429 and time_left > 0:
+            _logger.warning(
+                "API request to %s returned status code 429 (Rate limit exceeded). "
+                "Retrying in {sleep} seconds. "
+                "Will continue to retry 429s for up to {time_left} seconds."
+            ).format(sleep=sleep, time_left=time_left)
```

Comment:
```suggestion
            ).format(sleep=sleep, time_left=time_left, path=kwargs.get("url", "unknown path"))
```

(i think in practice we always pass the url)